build.gradle

```diff
@@ -730,7 +729,10 @@ ext.createDiffFile = { ->
         ':!services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceControllerService.java',
 
         // Other files that have tests but are not executed in the regular unit test task
-        ':!internal/alpini/*'
+        ':!internal/alpini/*',
+
+        // unit test for gRPC Transport Client is not straightforward, adding to exclusion list for now
```

Comment:
This needs to be addressed in the near future since it is critical to the GRPC path.
I assume it should be doable by passing some mock GRPC implementation.

Reply:
I agree. Currently, our gRPC service implementation is within `/services/venice-server`, we can move this to `venice-common` in order to create a mock implementation, but, the service implementation has a lot of dependencies within `venice-server` such as `RouterRequest` and `ReadResponse`. As such, moving this to the common folder is probably not a good idea due to these dependencies. I am looking into other solutions in order to create unit tests for this.

Reply:
Created an internal ticket to track the change.